controllers/controller.articles.js

```diff
@@ -37,3 +38,18 @@ exports.updateArticleById = (req, res, next) => {
       next(err);
     });
 };
+
+exports.getCommentsByArticleId = (req, res, next) => {
+  const { article_id } = req.params;
+  Promise.all([
+    fetchArticleById(article_id),
+    fetchCommentsByArticleId(article_id),
+  ])
+    .then((promises) => {
+      comments = promises[1];
+      res.status(200).send({ articleComments: comments });
+    })
+    .catch((err) => {
+      next(err);
+    });
+};
```

Comment:
I like how you've used promise.all here to check that the article exists at the same time as fetching the comments. Good work.